frontend/src/metabase/reference/databases/FieldDetailContainer.jsx

```diff
@@ -0,0 +1,83 @@
+/* eslint "react/prop-types": "warn" */
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import { connect } from 'react-redux';
+
+import FieldSidebar from './FieldSidebar.jsx';
+import SidebarLayout from 'metabase/components/SidebarLayout.jsx';
+import FieldDetail from "metabase/reference/databases/FieldDetail.jsx"
+
+import * as metadataActions from 'metabase/redux/metadata';
+import * as actions from 'metabase/reference/reference';
+
+import {
+    getDatabase,
+    getTable,
+    getField,
+    getDatabaseId,
+    getIsEditing
+} from '../selectors';
+
+const mapStateToProps = (state, props) => ({
+    database: getDatabase(state, props),    
+    table: getTable(state, props),    
+    field: getField(state, props),    
+    databaseId: getDatabaseId(state, props),
+    isEditing: getIsEditing(state, props)
+});
+
+const mapDispatchToProps = {
+    ...metadataActions,
+    ...actions
+};
+
+@connect(mapStateToProps, mapDispatchToProps)
+export default class FieldDetailContainer extends Component {
+    static propTypes = {
+        params: PropTypes.object.isRequired,
+        location: PropTypes.object.isRequired,
+        database: PropTypes.object.isRequired,
+        table: PropTypes.object.isRequired,
+        field: PropTypes.object.isRequired,
+        isEditing: PropTypes.bool
+    };
+
+    async fetchContainerData(){
+        await actions.rFetchDatabaseMetadata(this.props, this.props.databaseId);
+    }
+
+    async componentWillMount() {
```

Comment:
(React doesn't care at all about the return value of `componentWillMount`, `componentWillReceiveProps` etc)